pkg/generate.go

```diff
@@ -284,15 +284,15 @@ func generateDeploymentPatch(component gitopsv1alpha1.BindingComponentConfigurat
 		},
 	}
 
-	for _, env := range component.Env {
+	for _, env := range component.BaseEnvVar {
```

Comment:
```suggestion
	for _, env := range options.OverlayEnvVar {
```

since this is in patch

Reply:
Is my suggestion correct?

Reply:
`component.Env` was the original env var.  We're no just renaming it to `BaseEnvVar`.